site/en/docs/extensions/migrating/remote-hosted-code/index.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Replace remote hosted code
+subhead: 'Alternatives to remote hosted code in MV3'
+description: 'Different ways to migrate remote hosted code to MV3.'
+date: 2023-08-04
+---
+
+{% Partial 'extensions/mv3-support.md' %}
+
+This section provides an overview of different strategies for migrating MV2 extensions relying on remote hosted code to Manifest V3.
+
+Manifest V3 removes the ability for an extension to use [remotely hosted code][1], which presents security risks by allowing unreviewed code to be executed in extensions. With this change, an extension can only execute JavaScript that is included within its package and subject to review by the Chrome Web Store.
+
+## What is still possible? {: what-is-possible }
+
+There are still a few special cases in which executing remote hosted code and using <code>[eval][2]</code> is still possible:
+
+-   [Evaluate JavaScript in a sandboxed iframe][3].
+-   [Inject remote hosted stylesheets into a web page using insertCSS][4]
+-   For DevTool extensions:
```

Comment:
Hmm, there are automation use cases for chrome.debugger which don't fall under "DevTool" extensions. Maybe just "for extensions that use the chrome.devtools or chrome.debugger APIs"?

Reply:
rephrased - thanks